tests/test_app.py

```diff
@@ -315,6 +315,8 @@ def test_handle_event_update(
         assert mock_add_metadata.call_count == 1
         assert mock_generate_get_metadata_request_xml.call_count == 1
         assert handler.rabbit_client.send_message.call_count == 1
+        dt_string = mock_generate_get_metadata_request_xml.call_args[0][0]
+        assert type(datetime.strptime(dt_string, "%Y-%m-%dT%H:%M:%S.%f%z")) is datetime
```

Comment:
You can swap the `assert type(datetime.strptime(dt_string, "%Y-%m-%dT%H:%M:%S.%f%z")) is datetime` with `assert isinstance(datetime.strptime(dt_string, "%Y-%m-%dT%H:%M:%S.%f%z"), datetime)` to make deepcode ci bot happy :) 
The difference between type == and isinstance is explained here: https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance